Model/Api/Order.php

```diff
@@ -157,6 +166,15 @@ public function post($order, $action)
                     $orderForTransport = $this->_orderHelper->getOrderFulfillments();
                     $response = $transport->fulfillOrder($orderForTransport);
                     break;
+                case Api::ACTION_CHECKOUT_DENIED:
+                    if(get_class($order) == 'Magento\Quote\Model\Quote\Interceptor'){
+                        $this->_orderHelper->setOuote($order);
```

Comment:
No function setOuote.

Reply:
setOuote in defined in $this->_orderHelper

Reply:
So change name of this function to setQuote.

Reply:
I see, typo fixed.